000-vip-init.php

```diff
@@ -139,9 +139,9 @@
 	define( 'VIP_JETPACK_IS_PRIVATE', true );
 }
 
-// Jetpack Connection Pilot is enabled by default
+// Jetpack Connection Pilot is enabled by default on VIP Go environments
 if ( ! defined( 'VIP_JETPACK_AUTO_MANAGE_CONNECTION' ) ) {
-	define( 'VIP_JETPACK_AUTO_MANAGE_CONNECTION', true );
+	define( 'VIP_JETPACK_AUTO_MANAGE_CONNECTION', false !== VIP_GO_ENV );
```

Comment:
Let's use `WPCOM_IS_VIP_ENV` instead here since it is a proper boolean, per https://github.com/Automattic/vip-go-mu-plugins/pull/426.